tests/framework/checks/expects.go

```diff
@@ -0,0 +1,57 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright the KubeVirt Authors.
+ *
+ */
+
+package checks
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"kubevirt.io/client-go/kubecli"
+)
+
+const (
+	workerLabel                                  = "node-role.kubernetes.io/worker"
+	DescriptionTwoWorkerNodesWCPUManagerRequired = "at least two worker nodes with cpumanager are required for migration"
+)
+
+func ExpectAtLeastTwoWorkerNodesWithCPUManager(virtClient kubecli.KubevirtClient) {
+	ExpectAtLeastTwoNodes(GetWorkerNodesWithCPUManagerEnabled(virtClient), DescriptionTwoWorkerNodesWCPUManagerRequired)
+}
+
+// ExpectAtLeastTwoNodes uses gomega.Expect to verify that the given node list contains at least two elements. The
+// description is added to the default description via ginkgo.By
+func ExpectAtLeastTwoNodes(nodes []v1.Node, description string) bool {
```

Comment:
This does not seem to be exported anymore.

Reply:
I've inlined the func.